fenwick_tree/cpp/fenwick_tree.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+struct FenwickTree {
```

Comment:
Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]

*Origin: CPPLintBear, Section: `all.cpplint`.*